src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -67,45 +59,25 @@ public CaseSearchResult searchCases(
             + "the search results will contain only metadata by default (no case field data). To get case data in the "
             + "search results, please state the alias fields to be returned in the _source property for e.g."
             + " \"_source\":[\"alias.customer\",\"alias.postcode\"]",
-                  required = true)
+            required = true)
         @RequestBody String jsonSearchRequest) {
 
         Instant start = Instant.now();
 
-        rejectBlackListedQuery(jsonSearchRequest);
+        CrossCaseTypeSearchRequest request = elasticsearchQueryHelper.prepareRequest(
+            caseTypeIds,
+            UseCase.DEFAULT.getReference(),
```

Comment:
use case is a concept of the internal API only. It should not appear here.

Reply:
also, having a common helper to prepare both internal and external request will make our life more complicated in the future because it's likely the requests will have different requirements so you'll have code that doesn't make sense for one of the 2